diff --git a/data/org.pitivi.Pitivi.appdata.xml.in b/data/org.pitivi.Pitivi.appdata.xml.in
index eb2fb10..33ad0f4 100644
--- a/data/org.pitivi.Pitivi.appdata.xml.in
+++ b/data/org.pitivi.Pitivi.appdata.xml.in
@@ -36,7 +36,6 @@
     <kudo>Notifications</kudo>
   </kudos>
   <update_contact>pitivi@pitivi.org</update_contact>
-  <project_group>GNOME</project_group>
   <translation type="gettext">pitivi</translation>
   <developer_name>The Pitivi Team</developer_name>
 
