diff --git a/data/org.pitivi.Pitivi.appdata.xml.in b/data/org.pitivi.Pitivi.appdata.xml.in
index d1af6a344..0900f25af 100644
--- a/data/org.pitivi.Pitivi.appdata.xml.in
+++ b/data/org.pitivi.Pitivi.appdata.xml.in
@@ -313,7 +313,6 @@
           <li>Implementing many new features</li>
           <li>UI polish all over the place</li>
           <li>Refactoring pretty much the entire codebase</li>
-          <li>See http://developer.pitivi.org/releases/0-91.html for details</li>
         </ul>
       </description>
     </release>
