--- opencv/cmake/OpenCVCompilerOptions.cmake-orig	2023-03-22 17:20:20.986433861 -0400
+++ opencv/cmake/OpenCVCompilerOptions.cmake	2023-03-22 17:20:33.698668932 -0400
@@ -107,7 +107,6 @@
   endif()
   add_extra_compiler_option(-Werror=return-type)
   add_extra_compiler_option(-Werror=non-virtual-dtor)
-  add_extra_compiler_option(-Werror=address)
   add_extra_compiler_option(-Werror=sequence-point)
   add_extra_compiler_option(-Wformat)
   add_extra_compiler_option(-Werror=format-security -Wformat)
