--- a/setupext.py
+++ b/setupext.py
@@ -1751,7 +1751,10 @@ class BackendGtk3Agg(OptionalBackendPack
     def check_requirements(self):
         if 'TRAVIS' in os.environ:
             raise CheckFailed("Can't build with Travis")
-
+        # yoh: Builds of Debian packages often lead to a dead-lock here
+        #      As a workaround forcing build manually without a check
+        BackendAgg.force = True
+        return "ok"
         # This check needs to be performed out-of-process, because
         # importing gi and then importing regular old pygtk afterward
         # segfaults the interpreter.
